cvelib/cli.py

```diff
@@ -717,14 +729,16 @@ def users(ctx: click.Context, print_raw: bool) -> None:
 
 @cli.command()
 @click.pass_context
+@handle_cve_api_error
 def ping(ctx: click.Context) -> None:
     """Ping the CVE Services API to see if it is up."""
     cve_api = ctx.obj.cve_api
-    ok, error_msg = cve_api.ping()
+    ok, error = cve_api.ping()
 
     click.echo(f"CVE API Status — {cve_api.url}\n└─ ", nl=False)
     if ok:
         click.secho("OK", fg="green")
     else:
-        click.secho("ERROR:", bold=True, nl=False)
-        click.echo(f" {error_msg}")
+        # Let decorator format the error output
+        # Also makes command return correct status code on failure
```

Comment:
This line refers to a past state of the code, I think it can just be omitted :-)

Reply:
The intent here is to explain "why are we catching and then re-raising the exception?", which is just to allow the first output line to get printed ("CVE API Status") in addition to the error output. AKA "what happens if an exception gets raised here?" I can try to clarify the wording a bit so it's more about behavior and less about old vs. new code.